app/views/subscriptions/_subscribe_button.html.erb

```diff
@@ -1,27 +1,39 @@
 <%
   show_email_sub = true
   show_calendar_sub = (type == 'Event')
+  show_rss_sub = (type == 'Event')
 %>
 
-<% if show_calendar_sub || show_email_sub %>
+<% if show_calendar_sub || show_email_sub || show_rss_sub %>
   <div class="btn-group">
     <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
       Subscribe <span class="caret"></span>
     </button>
     <ul class="dropdown-menu">
       <% if show_email_sub %>
-          <li>
-            <a href="#" data-toggle="modal" data-target="#subscribe-email-modal">
-              <i class="fa fa-envelope-o"></i> Subscribe via email
-            </a>
-          </li>
+        <li>
+          <a href="#" data-toggle="modal" data-target="#subscribe-email-modal">
+            <i class="fa fa-envelope-o"></i> Subscribe via email
+          </a>
+        </li>
       <% end %>
       <% if show_calendar_sub %>
-          <li>
-            <a href="#" data-toggle="modal" data-target="#subscribe-calendar-modal">
-              <i class="fa fa-calendar-o"></i> Add to calendar
-            </a>
-          </li>
+        <li>
+          <a href="#" data-toggle="modal" data-target="#subscribe-calendar-modal">
+            <i class="fa fa-calendar-o"></i> Add to calendar
+          </a>
+        </li>
+      <% end %>
+      <% if show_rss_sub %>
+        <% rss_url = url_for(request.params.merge(format: :rss)) %>
```

Comment:
There's a method for extracting only the request params that are relevant to filtering, see:
https://github.com/ElixirTeSS/TeSS/blob/f807318a6fd55a82b492d9d2dff16043e25ff722/app/views/subscriptions/_subscribe_to_calendar_modal.html.erb#L13

Reply:
nice, done.